### ISSUE TITLE
Support different collations

### ISSUE THREAD

**Issue by DavidWiseman** [potatoqualitee/mssqlsuite#12]
Note: I've already done the work to support this change - I will submit a PR for your consideration.  

The action currently produces a SQL instance with the collation SQL_Latin1_General_CP1_CI_AS.  It would be useful to support other collations.

* Some apps are designed to work with a specific server collation
* An app might use SQL_Latin1_General_CP1_CI_AS for its collation but you want to perform testing with different or binary/case sensitive collations.  (My use case)

The action would work as before but support an **optional** parameter to pass in the collation. e.g.

````yml
        with:
          install: sqlengine, sqlclient, sqlpackage, localdb
          sa-password: c0MplicatedP@ssword
          show-log: true
          collation: Latin1_General_BIN
````

PS
Thank you for your work on this action.  I didn't think it was possible to run SQL on a Windows runner as Microsoft dropped support for SQL on Windows containers.  The examples I've seen use docker and only work on Linux.

I moved my app from .NET framework to .NET 6 - Linux is almost a possibility.  The main issue is the SSDT project requires msbuild which doesn't work on Linux.  It's starting to get support for dotnet build but it's not quite there yet.  I'd also have some other things to work out on Linux.  Having SQL on a Windows runner will allow me to create some useful GitHub actions now and revisit Linux support later.

If you are interested, my project is [DBA Dash](https://github.com/trimble-oss/dba-dash), an open-source monitoring tool for SQL Server.  I've done a PoC with this action and it works great.  Just need to create some good tests. 😊

**Comment by potatoqualitee** [comment]
released in 1.3 🥳

**Comment by DavidWiseman** [comment]
I've just added this workflow to DBA Dash, including the new change allowing me to switch to a binary collation (as it provides a better test)
Thank you.  🙏
